compiler/semstmts.nim

```diff
@@ -2363,7 +2363,8 @@ proc semProcAux(c: PContext, n: PNode, kind: TSymKind,
 proc determineType(c: PContext, s: PSym) =
   if s.typ != nil: return
   #if s.magic != mNone: return
-  #if s.ast.isNil: return
+  if s.ast.isNil: 
```

Comment:
CI output logs are gone, speculating maybe this would have helped:

```suggestion
  if s.ast.isNil and sfForward notin s.flags: 
```

May open new PR with this